am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GetAccessorsListIntegrationTest.java

```diff
@@ -26,26 +28,28 @@ void setUp() {
     @Test
     @SuppressWarnings("LineLength")
     void ifUserHasAccessShouldReturnUserIds() {
-        service.grantExplicitResourceAccess(createGrantForWholeDocument(resourceId, ACCESSOR_ID, READ_PERMISSION));
```

Comment:
How many times I need to ask to get it reverted?

Reply:
👍